app/components/providers/school-context.tsx

```
@@ -117,7 +117,7 @@ export function SchoolProvider({ children }: { children: ReactNode }) {
 
   useEffect(() => {
     fetchProviderSchools();
-  }, []);
+  }, [fetchProviderSchools]);
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)